docs/reference/generated/alert-dialog-root.json

```diff
@@ -12,7 +12,7 @@
       "description": "Whether the dialog is currently open."
     },
     "onOpenChange": {
-      "type": "((open: boolean, event: Event | undefined, reason: OpenChangeReason | undefined) => void)",
+      "type": "(open: boolean, event?: Event, reason?: AlertDialog.Root.OpenChangeReason) => void",
```

Comment:
This isn't 100% accurate and goes against https://www.typescriptlang.org/docs/handbook/2/functions.html#optional-parameters-in-callbacks but since it's just for the docs, we can leave it.

Reply:
Yeah, it looks shorter for the docs